docs_src/examples/Ch-ExamplesVirtualDeviceService.md

```diff
@@ -18,7 +18,7 @@ to simulate virtual resources.
 
 ## Introduction
 
-For information on the virtual device service see [virtual device](../microservices/device/virtual/Ch-VirtualDevice.md#) under the Microservices tab.
+For information on the virtual device service see [virtual device](../microservices/device/supported/device-virtual/Ch-VirtualDevice.md#) under the Microservices tab.
```

Comment:
Same changes needed for Levski PR for here and the others you found.

Reply:
They should be in sync with each other structure wise and stuff now.